addon/types/table.js

```diff
@@ -55,6 +59,19 @@ export default EmberObject.extend({
    */
   hooks: {},
 
+  init() {
+    this._super(...arguments);
+
+    if (this.options?.features?.localStorage) {
+      if (!this.options.name) {
+        throw new Error(
+          '[Hypertable] Trying to use localStorage store without a store name. Please set "options.name".'
+        );
+      }
+
+      this.set('store', new LocalStorageStore(this.options.name));
```

Comment:
i don't see that being used anywhere? when do you actually interact with the store?

Reply:
I think this will be done in the parent (project, component, etc) scope, where the component is actually used ?